### ISSUE TITLE
Model::loadMultiple does not work for non-tabular inputs with different models.

### ISSUE THREAD

**Issue by fortex** [yiisoft/yii2#8009]
I was very happy when I found the method to  load multiple models in basically one line.
I'm not sure if it is intended or not (probably, because it's documented that way), but it only works for tabular inputs (indexed form names e.g. Model[1][attribute]).

In my use case I have 3 different models in one single form, which get validated and saved together. 
Because they're different, I don't need tabular inputs. 

I understand for which use case this method was written for. But I would like to see something similar for non tabular inputs for instances of different model classes. I think my use case is almost as common as the one for tabular input is.

Thanks for reading :)

**Comment by mdmunir** [comment]
What you expected. What shortest code that you want?

**Comment by fortex** [comment]
At first I tried this with $models being an array of instances of different models

```php
if (Model::loadMultiple($models, $request->post())
                    && Model::validateMultiple($models)
                ) {
...
}
```

Which I liked until I realized, that this does not work. So, yeah. I expected it to work like that until I read the documentation and the body of the method.

Instead I have to do this now:

```php
if($modelOne->load($post) && $modelTwo->load($post) && $modelThree->load($post) && Model::validateMultiple($models)) {...}
```

Maybe ActiveForm should know about loading models via tabular input instead, just like it knows how to validate it for AJAX validation. (ActiveForm::validateMultiple)
